components/chunk_text/tests/chunk_text_test.py

```diff
@@ -3,11 +3,11 @@
 from src.main import ChunkTextComponent
 
 
-def test_transform():
+def test_transform__():
```

Comment:
Why the `__` suffix?

Reply:
PyCharm related bug. For some reasons I can't execute different tests with the same name in different folders.